templates/web/base/around/postcode_form.html

```diff
@@ -10,8 +10,9 @@
 
         <form action="[% c.uri_for('/around') %]" method="get" name="postcodeForm" id="postcodeForm" class="js-geolocate">
             <label for="pc">[% question %]:</label>
+            <p class="form-hint" id="pc-hint">[% tprintf(loc('e.g. ‘%s’ or ‘%s’'), c.cobrand.example_places) %]</p>
```

Comment:
I wonder if this might be better under the form? Perhaps to the left, with locate automatically to the right? Feels like part of heading where it is now, but might just be used to it.

Reply:
I think we leave it between the label and the input. That’s what we do everywhere else.